templates/docs/patterns/hero.md

```diff
@@ -304,3 +304,41 @@ For more complicated layouts utilise both columns of the 25/75% split and a nest
 Suru backgrounds are a common way to provide visual separation between hero and rest of the page.
 
 Check out the [Suru component documentation](/docs/patterns/suru) for more information on how to use it and relevant examples.
+
+## Higher Order Component
```

Comment:
"HOC" is not any specific version of hero.

Hero is a "hoc" (or pattern, as we should call it). So the whole hero documentation page is about it being a pattern. We should just make sure how existing examples for hero relate to new pattern designs, modify them if needed, only add new ones if really necessary (we already have quite a few of them).